### DISCUSSION TITLE
How to make extension for both VS2022 and 2019

### DISCUSSION THREAD

**Discussion by aglasencnik** [VsixCommunity/Community.VisualStudio.Toolkit#479] [Q&A]
Hi, I am wondering about how to make the extension to work in both VS2022 and 2019? Preferebly with the community toolkit package?
Any tutorials/examples out there?

Thanks for any info.

**Comment by reduckted** [comment]
Mads did a video tutorial on this a while ago when VS2022 was first announced. It's covered in the first half of this video (among lots of other things): https://www.youtube.com/live/L8b3kFriwCs?si=-ThJ_IojvS_5ma86

**Comment by ErikEJ** [comment]
SQLite Toolbox uses this toolkit and supports VS 2017, 2019 and 2022
